docs/README.md

```diff
@@ -1,72 +1,21 @@
-  They also provide background information and talk about design tradeoffs.
-- [Reference Guides](#reference-guides)
-  contain a thorough technical reference for complex topics.
-  They assume some overall understanding of surrounding subsystems.
-- [Recommended Practices](#recommended-practices)
-  suggests guidelines for writing code and diagnostics.
-- [Project Information](#project-information)
-  tracks continuous integration (CI), branching and release history.
-- [Evolution Documents](#evolution-documents)
-  includes proposals and manifestos for changes to Swift.
-- The [External Resources](#external-resources) section provides links to
-  valuable discussions about Swift development, in the form of talks
-  and blog posts.
-- The [Uncategorized](#uncategorized) section is for documentation which does
-  not fit neatly into any of the above categories. We would like to minimize
-  items in this section; avoid adding new documentation here.
+This page contains a curated index of the documentation found in the Swift
+compiler repository.
 
 Sometimes documentation is not enough.
 Especially if you are a new contributor, you might run into roadblocks
 which are not addressed by the existing documentation.
-Or they are addressed somewhere but you cannot find the relevant bits.
 If you are stuck, please use the [development category][] on the Swift forums
 to ask for help!
 
-Lastly, note that we are slowly moving towards a more structured form of
-documentation, inspired by the Django project [[1][Django-docs-1]]
-[[2][Django-docs-2]]. Hence parts of this page are aspirational
-and do not reflect how much of the existing documentation is written.
-Pull requests to clean up the [Uncategorized](#uncategorized) section,
-or generally fill gaps in the documentation are very welcome.
-If you would like to make major changes, such as adding entire new pieces of
-documentation, please create a thread on the Swift forums under the
-[development category][] to discuss your proposed changes.
-
 [development category]: https://forums.swift.org/c/development
-[Django-docs-1]: https://docs.djangoproject.com/
-[Django-docs-2]: https://documentation.divio.com/#the-documentation-system
-
-## Tutorials
-
-- [libFuzzerIntegration.md](/docs/libFuzzerIntegration.md):
-  Using `libFuzzer` to fuzz Swift code.
 
-## How-To Guides
+## Building the Toolchain
 
-- [FAQ.md](/docs/HowToGuides/FAQ.md):
-  Answers "How do I do X?" for a variety of common tasks.
-- [FirstPullRequest.md](/docs/HowToGuides/FirstPullRequest.md):
-  Describes how to submit your first pull request. This is the place to start
-  if you're new to the project!
 - [GettingStarted.md](/docs/HowToGuides/GettingStarted.md):
   Describes how to set up a working Swift development environment
   for Linux and macOS, and get an edit-build-test-debug loop going.
-- [DebuggingTheCompiler.md](/docs/DebuggingTheCompiler.md):
-  Describes a variety of techniques for debugging.
 - Building for Android:
```

Comment:
I don't think this should go in the `Building the Toolchain` section, if I understand the documents correctly, they explain how to build Swift programs on Android and OpenBSD, not how to build the toolchain on these platforms.